examples/sftp-client-send-file/sftp_client_send_file.metatags

```diff
@@ -1,2 +1,2 @@
-description: This is a BBE on getting and putting file content on a remote SFTP server.
+description: This example demonstrates putting file content on a remote SFTP server.
```

Comment:
```suggestion
description: This example demonstrates adding file content to a remote SFTP server.
```
Check if the technical meaning got affected by the suggested change.

Reply:
The api is defined as `put`. So I think it's better if we say `putting`